indico/modules/events/editing/client/js/management/editable_type/EditableList.module.scss

```diff
@@ -0,0 +1,88 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@import 'base/palette';
+
+$border: 1px solid $pastel-gray;
+
+.editable-topbar {
+    display: flex;
+    width: 1000px;
+    justify-content: space-between;
+}
+
+.editable-list {
+    .table {
+        font-family: 'Muli', 'Helvetica Neue', Arial, Helvetica, sans-serif;
+        margin-top: 20px;
+        margin-bottom: 50px;
+    }
+
+    .rowcolumn-tooltip {
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        overflow: hidden;
+    }
+
+    :global(.ReactVirtualized__Table__Grid) {
+        outline: none;
+    }
+
+    :global(.ReactVirtualized__Table__headerRow) {
+        font-weight: 700;
+        text-transform: uppercase;
+        display: flex;
+        border-top: $border;
+        border-bottom: $border;
+    }
+
+    :global(.ReactVirtualized__Table__row) {
+        display: flex;
+        border-bottom: $border;
+    }
+
+    :global(.ReactVirtualized__Table__headerTruncatedText) {
+        display: inline-block;
+        max-width: 100%;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow: hidden;
+    }
+
+    :global(.ReactVirtualized__Table__headerColumn),
+    :global(.ReactVirtualized__Table__rowColumn) {
+        margin-right: 10px;
+        align-items: center;
+        display: flex;
+    }
+
+    :global(#status.ReactVirtualized__Table__headerColumn) {
+        justify-content: center;
```

Comment:
By adding this css you have the STATUS column spaced evenly in the center (even the header) but the rest of the columns are spaced on the left side. I would suggest to remove this all together and just add some spacing between the `StateIndicator` and the state text.

Before: 
![image](https://user-images.githubusercontent.com/17234430/80730419-d99af880-8b09-11ea-882f-0bf7f77d18b2.png)
After:
![image](https://user-images.githubusercontent.com/17234430/80730609-12d36880-8b0a-11ea-9de3-0f1f61f77cbf.png)

Also, that could be avoided if the status text was part of the `StateIndicator`